elementary/monitor/dbt_project/macros/update_sent_alerts.sql

```diff
@@ -1,8 +1,8 @@
-{% macro update_sent_alerts(alert_ids, table_name) %}
+{% macro update_sent_alerts(alert_ids, sent_at, table_name) %}
     {% if alert_ids %}
         {% set update_sent_alerts_query %}
-            UPDATE {{ ref(table_name) }} set alert_sent = TRUE
-            WHERE alert_id IN {{ elementary.strings_list_to_tuple(alert_ids) }} and alert_sent = FALSE and
+            UPDATE {{ ref(table_name) }} set suppression_status = 'sent', sent_at = {{ "'" ~ sent_at ~ "'" }}, alert_sent = TRUE
```

Comment:
Can you use `elementary.quote()` instead?
Also, I think we should try to avoid this kind of string formatting in general as it's really ugly and use Python's string formatting (`"'{}'".format("...")`) instead.

Reply:
Agree!